nemo/collections/asr/data/feature_to_label.py

```diff
@@ -137,3 +255,82 @@ class FeatureToSeqSpeakerLabelDataset(_FeatureSeqSpeakerLabelDataset):
 
     def _collate_fn(self, batch):
         return _feature_collate_fn(batch)
+
+
+class FeatureToLabelDataset(Dataset):
+    """
+    Dataset that loads tensors via a json file containing paths to feature files and their labels. 
+    Each new line is a different sample. Example below:
+    and their target labels. JSON files should be of the following format:
+        {"feature_filepath": "/path/to/audio_feature.pt", "label": "1"} \
+        ...
+        {"feature_filepath": "/path/to/audio_feature.pt", "label": "0"} 
+    Args:
+        manifest_filepath (str): Dataset parameter. Path to JSON containing data.
+        labels (Optional[list]): Dataset parameter. List of unique labels collected from all samples.
+        augmentor (Optional): feature augmentation
+
+    """
+
+    ZERO_LEVEL_SPEC_DB_VAL = -16.635  # Log-Melspectrogram value for zero signal
```

Comment:
can we make this MASK_VAL and make it an input argument, to enable experimenting with different values?